app-editors/neovim/files/neovim-0.8-msgpack-6.0.0-fix.patch

```diff
@@ -0,0 +1,25 @@
+# Closes: https://bugs.gentoo.org/903629
+# Contributed by Olivier Huber <oli.huber@gmail.com>
```

Comment:
Please always include a link to the relevant upstream commits.

Reply:
I'll put it in so we can get this merged.

Reply:
Noted and will keep in mind